types/aframe/tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
That's understandable. I can make the rest of the changes you've requested, but I'd like to be able to keep these integration tests to help me (and other devs) track changes to aframe's API. The definition tests are OK, but it won't inform us if there's a change to the API between versions. I understand the tests wont be run by the CI.